data/stats.yaml

```
@@ -1,7 +1,7 @@
 - date: 2023-05-01
-  services: 650
-  terms: 1300
+  services: 712
+  terms: 1429
```

These are currently hardcoded with a `+`, so either remove the `+` from the template or round it down.
```suggestion
  services: 700
  terms: 1400
```